Hw5.sql

```diff
@@ -0,0 +1,40 @@
+use baseball;
```

Comment:
SQL works well. Not sure why when you call the table in python all the column labels disappear. Otherwise, try getting some comparative stats between teams as Julien mentioned in class.